src/components/activity/ActivityDirectivesTable.svelte

```diff
@@ -117,6 +130,29 @@
     completeColumnDefs = [activityErrorColumnDef, ...(columnDefs ?? []), activityActionColumnDef];
   }
 
+  onMount(() => {
+    document.addEventListener(`keydown`, onKeyDown);
```

Comment:
I'm getting an error about document not being defined when i switch to this branch and svelte-kit tries to do the first server side render.
```
ReferenceError: document is not defined
    at /Users/aplave/code/aerie-ui/src/components/activity/ActivityDirectivesTable.svelte:138:5
```

An alternative here would be to use:
```
<svelte:window on:keydown={onKeydown} />
```
instead which effectively does what you're doing with mount/destroy but shouldn't have the same issue with SSR.